### ISSUE TITLE
Neither global nor user snippets works for yang files when yangster is installed. 

### ISSUE THREAD

**Issue by hniska** [theia-ide/yang-vscode#52]
Latest vscode (1.65.2) and yang-vscode (2.0.6)

I have a snippet with a prefix set to leaf, when typing leaf I get this error "name must not be falsy: Error: name must not be falsy"

```
[2022-03-23 16:10:26.386] [renderer1] [info] [KeybindingService]: / Received  keydown event - modifiers: [], code: KeyL, keyCode: 76, key: l
[2022-03-23 16:10:26.387] [renderer1] [info] [KeybindingService]: | Converted keydown event - modifiers: [], code: KeyL, keyCode: 42 ('L')
[2022-03-23 16:10:26.388] [renderer1] [info] [KeybindingService]: | Resolving [KeyL]
[2022-03-23 16:10:26.388] [renderer1] [info] [KeybindingService]: \ From 1 keybinding entries, no when clauses matched the context.
[2022-03-23 16:10:26.436] [renderer1] [info] [KeybindingService]: / Received  keydown event - modifiers: [], code: KeyE, keyCode: 69, key: e
[2022-03-23 16:10:26.437] [renderer1] [info] [KeybindingService]: | Converted keydown event - modifiers: [], code: KeyE, keyCode: 35 ('E')
[2022-03-23 16:10:26.438] [renderer1] [info] [KeybindingService]: | Resolving [KeyE]
[2022-03-23 16:10:26.438] [renderer1] [info] [KeybindingService]: \ No keybinding entries.
[2022-03-23 16:10:26.493] [renderer1] [info] [KeybindingService]: / Received  keydown event - modifiers: [], code: KeyA, keyCode: 65, key: a
[2022-03-23 16:10:26.496] [renderer1] [info] [KeybindingService]: | Converted keydown event - modifiers: [], code: KeyA, keyCode: 31 ('A')
[2022-03-23 16:10:26.497] [renderer1] [info] [KeybindingService]: | Resolving [KeyA]
[2022-03-23 16:10:26.498] [renderer1] [info] [KeybindingService]: \ From 1 keybinding entries, no when clauses matched the context.
[2022-03-23 16:10:26.643] [renderer1] [info] [KeybindingService]: / Received  keydown event - modifiers: [], code: KeyF, keyCode: 70, key: f
[2022-03-23 16:10:26.643] [renderer1] [info] [KeybindingService]: | Converted keydown event - modifiers: [], code: KeyF, keyCode: 36 ('F')
[2022-03-23 16:10:26.644] [renderer1] [info] [KeybindingService]: | Resolving [KeyF]
[2022-03-23 16:10:26.644] [renderer1] [info] [KeybindingService]: \ From 1 keybinding entries, no when clauses matched the context.
[2022-03-23 16:10:26.998] [renderer1] [error] name must not be falsy: Error: name must not be falsy
	at Function.validate (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:76:103993)
	at new n (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:76:103942)
	at new n (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:76:90384)
	at O (/Users/hniska/.vscode/extensions/typefox.yang-vscode-2.0.6/extension/pack/extension.js:1:152505)
	at Array.map (<anonymous>)
	at Object.asDocumentSymbols (/Users/hniska/.vscode/extensions/typefox.yang-vscode-2.0.6/extension/pack/extension.js:1:156580)
	at /Users/hniska/.vscode/extensions/typefox.yang-vscode-2.0.6/extension/pack/extension.js:1:34277
	at async f.provideDocumentSymbols (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:85:91158)
```

**Comment by dhuebner** [comment]
@hniska 
Could you confirm that this problem exists in the last released version 2.0.2?

**Comment by hniska** [comment]
Hi, 

Ill close this, after a java update it now works.
